azure/iac/master-compliance-test.json

```diff
@@ -2674,6 +2674,130 @@
                             ]
                         }
                     ]
+                },
+                {
+                    "masterTestId": "TEST_sql_servers_encryption",
+                    "masterSnapshotId": [
+                        "ARM_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(sql_servers_encryption.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0111-ARM",
+                            "eval": "data.rule.serverKeyType",
+                            "message": "data.rule.serverKeyType_err",
+                            "remediationDescription": "Make sure SQLServer/encryptionProtector has ( properties.serverKeyType ='AzureKeyVault' )",
+                            "remediationFunction": "PR-AZR-0111-ARM.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "Ensure SQL server's TDE protector is encrypted with Customer-managed key",
+                    "description": "Customer-managed key support for Transparent Data Encryption (TDE) allows user control of TDE encryption keys and restricts who can access them and when. Azure Key Vault, Azure’s cloud-based external key management system is the first key management service where TDE has integrated support for Customer-managed keys. With Customer-managed key support, the database encryption key is protected by an asymmetric key stored in the Key Vault. The asymmetric key is set at the server level and inherited by all databases under that server.",
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": [
+                                "Best Practices"
+                            ],
+                            "service": [
+                                "arm"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "masterTestId": "TEST_postgreSQL_1",
+                    "masterSnapshotId": [
+                        "ARM_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(postgreSQL.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0115-ARM",
+                            "eval": "data.rule.geoRedundantBackup",
+                            "message": "data.rule.geoRedundantBackup_err",
+                            "remediationDescription": "Make sure PostgreSQL has ( properties.storageProfile.geoRedundantBackup = 'Enabled' )",
```

Comment:
**See my validation message at line 2496 (for id PR-AZR-0109-ARM).** make the description format like this one.